src/php/binaries.rs

```diff
@@ -31,7 +31,12 @@ pub(crate) fn all() -> HashMap<PhpVersion, PhpBinary> {
 
     merge_binaries(binaries_from_dir(PathBuf::from("/usr/bin")), &mut binaries);
     merge_binaries(binaries_from_dir(PathBuf::from("/usr/sbin")), &mut binaries);
-    merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/Cellar/php/*/")), &mut binaries);
+
+    if cfg!(target_os = "macos") {
+        merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/Cellar/php/*/bin")), &mut binaries);
+        merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/Cellar/php@*/*/bin")), &mut binaries);
+        merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/php*/bin")), &mut binaries);
+    }
```

Comment:
There's no need to make it mac-specific, since there can be binaries there anyway, especially considering the fact that Homebrew can be installed on Linux.
So let's check them anyway :)

```suggestion
    merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/Cellar/php/*/bin")), &mut binaries);
    merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/Cellar/php@*/*/bin")), &mut binaries);
    merge_binaries(binaries_from_dir(PathBuf::from("/usr/local/php*/bin")), &mut binaries);
```

Plus, I'm working on the config file, so the lookup will only be done once, so there's no problem with performances here. I'll add a `--refresh-binaries` option later so the command will be fast all the time, and if the users install a new PHP version, they can refresh the whole list. I'll add this to the roadmap :)

Reply:
Oh, didn't knew that Homebrew existed outside Apple world.
Will push an update after you completed your review.

Reply:
Solved